gemfiles/rails_8.0_ruby_3.4.gemfile

```
@@ -0,0 +1,28 @@
+# This file was generated by Appraisal
```

RuboCop (1.69.1)

[convention] Missing frozen string literal comment.

(Style/FrozenStringLiteralComment)